README.md

```diff
@@ -1,4 +1,4 @@
-![3ds Max](https://img.shields.io/badge/3ds%20Max-2024%20%7C%202023%20%7C%202022%20%7C%202021-blue)
+![3ds Max](https://img.shields.io/badge/3ds%20Max-2025%20%7C%202024%20%7C%202023%20%7C%202022%20%7C%202021-blue)
```

Comment:
```suggestion
![3ds Max](https://img.shields.io/badge/3ds%20Max-2025%20%7C%202024%20%7C%202023%20%7C%202022-blue)
```

Reply:
latest + 4 versions before?

Reply:
Latest + 3 before.

I kept 2021 when we added 2024. But now, we are adding 2025, we should remove it.